### ISSUE TITLE
About CRIT WOTLK 

### ISSUE THREAD

**Issue by dxxa** [dalaranwow/dalaran-wow#3253]
I did some testing to control the crit on this server, I saw that in my opinion is totally mistaken, .in WotLK crit to be calculated ( x1.25 or x1.50), the screen below you can see that the crit on this server is calculated (x2,0) as it is in cataclism / mop / wod, below I put the screen:

# **_Vampiric Touch_**
![immagine crit](https://cloud.githubusercontent.com/assets/18446530/14905393/66c5d65c-0db0-11e6-955c-5c2cdd81ebc0.png)

# **_Shadow Word: Pain_**
![immagine crit 2](https://cloud.githubusercontent.com/assets/18446530/14905424/a98f6098-0db0-11e6-903a-1adf7f34a0e1.png)

# **_Devouring Plague_**

![immagine crit 3](https://cloud.githubusercontent.com/assets/18446530/14905444/e962c232-0db0-11e6-9e85-57ffd6cd672f.png)

# **_Living Bomb_**

![immagine crit living](https://cloud.githubusercontent.com/assets/18446530/14905673/53289d34-0db3-11e6-9cc8-f189745869ee.png)

**Comment by Mike-Hunt** [comment]
Hi! Spells in Wrath in general have a base +150% crit damage bonus, you're right. What you're missing is that talents and meta-gems modify this to give higher results. From the numbers in your screenshots, I am assuming both your priest and mage are talented for the crit damage boost in their trees (Shadow Power and Burnout respectively) and have a Chaotic Skyflare Diamond equipped. 
CSD actually increases the end value of your crit by more than 3% because it double dips with crit damage boosting talents (this was the case in retail as well), I'll show how.
For the Fire mage's Living Bomb: 944 x 1.5(normal crit) x 1.03(CSD) = 1458.48.
1458.48 - 944 = 514.48 = your crit damage. 
This crit damage is then mulitplied by burnout (+50% crit damage), so you get 944 + 514.48 x 1.5 = 1715 = the damage in your screenshot.

For the Shadow Priest, the calculations are similar, except instead of a 50% crit damage boost via talents they have 100%. Combining that with the CSD, you can simplify it to 209% end crit damage bonus.
For VT: 1478 x 2.09 = 3089.2, you got 3088.
For SW:P: 947 x 2.09 = 1979.23, you got 1977
For DP: 801 x 209 = 1674.09, you got 1673. The minor differences are due to rounding.

So basically, this report is invalid, you just didn't account for talents/your meta-gem. In the future, if you have issues like this you're better off researching on elitistjerks/wowhead forums/guild mates etc to make sure you haven't missed something very simple before you clutter the bugtracker.

**Comment by Golganeth** [comment]
Inaccurate and invalid report. Thank you for the input @Mike-Hunt.

Closed

**Comment by Lukas45** [comment]
I don't know if it's an invalid report though, since the only talent I can find that affects crit damage for shadow priests is Shadow Power (http://wotlk.openwow.com/spell=33225) - but according to OpenWoW it only affects Mind Flay, Mind Blast and Shadow Word: Death - Unless that's a tooltip error on their end.

But as the combat logs show, those DoTs have 200% crit damage.

Unless I missed another talent.
